rllib/agents/qmix/qmix_policy.py

```diff
@@ -530,29 +537,32 @@ def _unpack_observation(self, obs_batch):
                 or None if it is not in the batch
         """
 
-        unpacked = _unpack_obs(
-            np.array(obs_batch, dtype=np.float32),
-            self.observation_space.original_space,
-            tensorlib=np,
-        )
+        if not self.config["_disable_preprocessor_api"]:
```

Comment:
maybe we should add this to the default config dict for QMix and set its default value to True, and provide some detailed instruction of what we are disabling here (_unpack_obs)?
is there a scenario we still want this?
QMix is the only agent that uses _unpack_obs preprocessor, so it will be exciting if we can simply get rid of it ...